src/components/views/articles/SingleArticle.js

```diff
@@ -10,6 +12,7 @@ import { Link } from "react-router-dom";
 
 import * as actionGenerators from "../../../actions/articlesActions";
 import HeaderComponent from "../../containers/headers/index";
+import * as tagSearching from "../../../actions/tagSearchActions";
```

Comment:
I think it's better to have named it tagSearchingActions.

Reply:
@bettblake08 let me rename it to fit its function